dm/syncer/syncer.go

```diff
@@ -123,7 +123,17 @@ const (
 type Syncer struct {
 	sync.RWMutex
 
-	tctx *tcontext.Context
+	tctx *tcontext.Context // this ctx only used for logger.
+
+	// this ctx derives from a background ctx and was initialized in s.Run, it is used for some background tasks in s.Run
+	// when this ctx cancelled, syncer will shutdown all background running jobs and not wait transaction end
+	runCtx    *tcontext.Context
+	runCancel context.CancelFunc
+	// this ctx only used for syncDML and syncDDL and only cancelled when ungraceful stop
+	syncTCtx   *tcontext.Context
```

Comment:
I mean TCtx is ugly 😂 

BTW, by your comment for these two context, one is "when this ctx cancelled, syncer will shutdown all background running jobs and not wait transaction end" other is "only cancelled when ungraceful stop", I can't get the difference of them.

Reply:
renamed to syncCtx  and updated the comment about these two ctx in https://github.com/pingcap/tiflow/pull/4035/commits/705c228dd22049a3499c4171ff5ddbbf01f12b10, dose this help?